From fd1bb72f32b098abd2b13f9d03bc754071b0cfba Mon Sep 17 00:00:00 2001
From: Heinrich Schuchardt <xypron.glpk@gmx.de>
Date: Sun, 22 Jul 2018 11:34:34 +0200
Subject: [PATCH 1/1] dm: sysreset: x86: missing build dependency

make qemu-x86_defconfig
with additional

  #CONFIG_SYSRESET is not set

results in

  LD      u-boot
  lib/built-in.o: In function `efi_reset_system_boottime':
  lib/efi_loader/efi_runtime.c:122: undefined reference to `do_reset'

Fixes: fabb2b4c7f28 ("dm: sysreset: x86: Add a sysreset driver")
Signed-off-by: Heinrich Schuchardt <xypron.glpk@gmx.de>
---
 drivers/sysreset/Kconfig | 12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

diff --git a/drivers/sysreset/Kconfig b/drivers/sysreset/Kconfig
index 9b2fda4d25..a5a58e6d83 100644
--- a/drivers/sysreset/Kconfig
+++ b/drivers/sysreset/Kconfig
@@ -36,6 +36,12 @@ config SYSRESET_PSCI
 	  Enable PSCI SYSTEM_RESET function call.  To use this, PSCI firmware
 	  must be running on your system.
 
+config SYSRESET_X86
+	bool "Enable support for x86 processor reboot driver"
+	depends on X86
+	help
+	  Reboot support for generic x86 processor reset.
+
 endif
 
 config SYSRESET_SYSCON
@@ -51,10 +57,4 @@ config SYSRESET_WATCHDOG
 	help
 	  Reboot support for generic watchdog reset.
 
-config SYSRESET_X86
-	bool "Enable support for x86 processor reboot driver"
-	depends on X86
-	help
-	  Reboot support for generic x86 processor reset.
-
 endmenu
-- 
2.18.0

